src/main/cliapp/src/service/BaseAuthService.js

```diff
@@ -6,7 +6,11 @@ export class BaseAuthService {
 
 	constructor() {
 		let oktaTokenStorage = localStorage.getItem('okta-token-storage');
+
+		if(!oktaTokenStorage) oktaTokenStorage = JSON.stringify({}); 
```

Comment:
@adamgibs does this check for an empty string also?

Reply:
Nope, only if exists or not. Should it also check for an empty string?

Reply:
I would think so...actually what would be better is to catch the error of `JSON.stringify` and if there is an error set it to `JSON.stringify({});`